src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/ui/CommandProcessorPanel.java

```diff
@@ -60,7 +60,7 @@ public CommandProcessorPanel(CommandProcessor cp) {
 
         editor = new JTextArea();
         editor.setDocument(new EditableAtEndDocument());
-        editor.setFont(GraphicsUtilities.lookupFont("Courier"));
+        editor.setFont(new Font(Font.MONOSPACED, Font.PLAIN, GraphicsUtilities.FONT_SIZE));
```

Comment:
Maybe this can be changed to `GraphicsUtilities.getMonospacedFont()` so there's no need to export FONT_SIZE?

Reply:
> Maybe this can be changed to `GraphicsUtilities.getMonospacedFont()` so there's no need to export FONT_SIZE?

Yeah, it is the point what I thought too, but I didn't do so because it is one liner.
However I re-think about it from your comment, so I pushed new commit to add `GraphicsUtilities.getMonospacedFont()`.